### PR TITLE
LRAC 8015 | master

### PR DESCRIPTION
**Pull request by calvinctvo**
https://issues.liferay.com/browse/LRAC-8015

### CODE REVIEW

**Comment by calvinctvo** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsList.testcase [hunk 1]
```diff
@@ -83,4 +83,58 @@ definition {
...
+
+		Pause(locator1 = "5000");
+
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "userea@liferay.com",
+			userLoginFullName = "userfn userln");
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "Andre DXP 7.4");
```
Comment (joshitagaki) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsList.testcase:107]: The property name should be Liferay DXP?

**Comment by calvinctvo** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsList.testcase (hunk 1 above)
Comment (calvinctvo) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsList.testcase]: thank you @joshitaki

**Comment by calvinctvo** [comment]
ci:forward

**Pull request closed without merging**